### ISSUE TITLE
Add gaffer-slider integration tests

### ISSUE THREAD

**Issue by ctas582** [gchq/gaffer-tools#52]

**Comment by ctas582** [comment]
Being able to run Accumulo Store ITs against a gaffer-slider instance will require Gaffer/issues#686 to be fixed.

**Comment by ctas582** [comment]
Running AbstractStoreITs against a gaffer-slider instance is working.
Being able to add AccumuloStoreITs to the test suite is blocked by gchq/Gaffer#686.
I've logged issue #71 so that this can be enhanced with the AccumuloStoreITs at a later date.

**Comment by p013570** [comment]
Merged into develop.
